assets/js/code-snippets.js

```diff
@@ -0,0 +1,5 @@
+$(function() {
```

Comment:
Is there a chance there will be any more JS code needed for the style guide itself like this? In that case, I'd call this file styleguide.js, otherwise this is fine as is.